akka-actor/src/main/mima-filters/2.6.13.backwards.excludes/30065-timers-initialDelay.excludes

```diff
@@ -0,0 +1,22 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.actor.TimerScheduler.startTimerWithFixedDelay")
```

Comment:
Worth adding a comment why these are okay to exclude. i.e. that some are public APIs but are DoNotInherit and some are private

Reply:
Good hint, I've added comments to document this